src/components/Card/card.module.scss

```diff
@@ -0,0 +1,130 @@
+.card {
+    --bg-color: white;
+
+    box-shadow: $shadow-object-m;
+    background-color: var(--bg-color);
+    border-radius: $corner-radius-xl;
+    color: black;
+    display: block;
+    font-family: $octuple-font-family;
+    gap: $space-m;
+    padding: $space-ml;
+    width: 360px;
```

Comment:
we should not be so restrictive. this should be a block element and implementation would dictate the size/scale of